eval/op.go

```diff
@@ -242,6 +258,10 @@ func binOp(op token.Token, x, y interface{}) (interface{}, error) {
 			case UntypedInt:
 				z := big.NewInt(0)
 				return UntypedInt{z.Sub(x.Int, y.Int)}, nil
+			case UntypedComplex:
+				re := big.NewFloat(0)
+				xf := big.NewFloat(float64(x.Int.Int64()))
+				return UntypedComplex{re.Sub(xf, y.Real), big.NewFloat(0)}, nil
```

Comment:
Why are you ignoring the Imag component of y?

Reply:
copy-pasta...
done.